src/css/_print.scss

```diff
@@ -0,0 +1,5 @@
+@media print {
+  .video-js *:not(.vjs-tech):not(.vjs-poster) {
+    visibility:hidden;
```

Comment:
Using visibility:hidden guarantees that we don't have a reflow of the page. 'you print what you see' in terms of dimensions. When using display:none; you are actively removing things from the canvas, which can influence the dimensions and behavior of other elements.

In reality, it probably won't matter much unless someone writes a 'bad' plugin.